skimage/segmentation/_quickshift.pyx

```diff
@@ -112,7 +113,7 @@ def quickshift(image, ratio=1.0, kernel_size=5, max_dist=10,
                             dist += (current_pixel_p[channel] -
                                      image_c[r_, c_, channel])**2
                         dist += (r - r_)**2 + (c - c_)**2
-                        densities[r, c] += exp(-dist / (2 * kernel_size**2))
+                        densities[r, c] += exp(-dist / (2 * kernel_size_c**2))
```

Comment:
I am fairly certain that the compiler is smart to optimize, but why not go all the way and make it `inv_neg_kernel_size_sqr = -0.5 / kernel_size**2`? The loss in precision shouldn't matter too much in this case I assume.